internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -354,6 +354,39 @@ type networkLoadBalancerConfig struct {
 	appDNSName           *string
 }
 
+func convertEnvSecurityGroupCfg(mft *manifest.Environment) (*template.SecurityGroupConfig, error) {
+	securityGroupConfig, isSecurityConfigSet := mft.EnvSecurityGroup()
+	if !isSecurityConfigSet {
+		return nil, nil
+	}
+	var ingress = make([]template.SecurityGroupRule, len(securityGroupConfig.Ingress))
+	var egress = make([]template.SecurityGroupRule, len(securityGroupConfig.Egress))
+	for idx, ingressValue := range securityGroupConfig.Ingress {
+		ingress[idx].IpProtocol = ingressValue.IpProtocol
+		ingress[idx].CidrIP = ingressValue.CidrIP
+		if fromPort, toPort, err := ingressValue.Ports(); err != nil {
+			return nil, err
+		} else {
+			ingress[idx].ToPort = toPort
+			ingress[idx].FromPort = fromPort
+		}
```

Comment:
nit:
```suggestion
		if fromPort, toPort, err := ingressValue.Ports(); err != nil {
			return nil, err
		} 
		ingress[idx].ToPort = toPort
		ingress[idx].FromPort = fromPort
```

Reply:
I have not defined `fromPort` and `toPort` outside of the `if` scope so `fromPort` and `toPort` will only work if I write them in either `if` or `else`. But I am happy to declare them separately outside of the if scope and then do what you are suggesting. Let me know if you want me to do that.